### PR TITLE
🔨 BIGMAP: minor fixes

### LINKED ISSUE #295: suggestions for code/computer setup
* don't use one-letter options in a tutorial - or you will have to explain them later. if you use the full version, they are self-explanatory. i.e. instead of
```
verdi code setup -L add --computer=localhost -P arithmetic.add --remote-abs-path=/bin/bash -n
```
use
```
verdi code setup --label add --computer localhost --input-plugin arithmetic.add --remote-abs-path /bin/bash --non-interactive
```

 * typo `\bin\bash` in instructions

 * consider removing the `verdi computer show localhost` - not sure why people need to worry about this at this stage
   this will also save you the "one set of pks for one type of entity discussion"

### LINKED ISSUE #296: minor fix
In case that your nodes’ PKs are different and you don’t remember the PK of the output node from the previous calculation
=> 
In case you don’t remember the PK of the output node from the previous calculation

(there is no pk shown)

### LINKED ISSUE #297: add one sentence on shutting down ngrok + rest api
Before moving to bands export, add one sentence on closing the ngrok + terminal + stopping the REST API

### PR DESCRIPTION
**Pull request by mbercx**
Fixes #295 
Fixes #296 
Fixes #297

### CODE REVIEW

File: docs/pages/2020_BIGMAP/sections/basics.rst [hunk 1]
```diff
@@ -410,7 +411,7 @@ Now, exit the IPython shell and once more check for *all* processes:
 
 .. code-block:: console
 
-    $ verdi process list -a
+    $ verdi process list --all
```
Comment (ramirezfranciscof) [inline docs/pages/2020_BIGMAP/sections/basics.rst:414]: I mean...ok but why? `-a` should work fine no?

File: docs/pages/2020_BIGMAP/sections/qe.rst [hunk 2]
```diff
@@ -407,7 +417,7 @@ Let's have a look at the groups we've imported from the archive above, using the
 
 .. code-block:: console
 
-    $ verdi group list -C
+    $ verdi group list --count
```
Comment (ramirezfranciscof) [inline docs/pages/2020_BIGMAP/sections/qe.rst:420]: Idem here

**Review by ramirezfranciscof — APPROVED** [review]
All good, it can go as it is, I just have two questions regarding why do you changed to the long versions of `verdi process list --all` and `verdi group list --count`.

File: docs/pages/2020_BIGMAP/sections/basics.rst (hunk 1 above)
Comment (mbercx) [reply docs/pages/2020_BIGMAP/sections/basics.rst:414]: See #295

**Pull request merged**